src/components/run/InterpretationLog.tsx

```
@@ -250,69 +562,246 @@ export const InterpretationLog: React.FC<InterpretationLogProps> = ({ isOpen, se
+                      >
+                        {captureListData.map((listItem, index) => {
+                          const isEditing = editingListName === listItem.id;
+                          const isActive = activeListTab === index;
+
+                          return (
+                            <Tooltip
+                              title="Double click to edit captured list name"
+                              arrow
+                              placement="top"
+                            >
+                              <Box
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)